### PR TITLE
feat: Updated Membership section

### LINKED ISSUE #4054: [Handbook] Update Information regarding Membership to the Github organizations
### Current Behavior

The current information does not cover how the membership is awarded to an individual. 

### Desired Behavior

The mentioned information for the Membership in the [handbook](https://layer5.io/community/handbook/recognition) has to be rephrased and updated with new updates in the organization. The number of organizations should be 5 instead of four now.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Description**

- This PR fixes #4054
- Have Rephrased the Paragraph, mentioning the significance of membership, and how it does not just revolve around code contributions.
- extended it by mentioning what are things that are expected to become one of the members.
 
**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8a977255f1d7ad38fe40b915c249a0a07b11bd12 at: https://642c67040c38b821f07093e2--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f8233425e6136a0af2dfd97199dabf8dd9453020 at: https://642d7431b993e43177da3bbc--layer5.netlify.app

**Review by Abhishek-kumar09 — APPROVED** [review]

File: src/sections/Community/Handbook/recognition.js [hunk 1]
```diff
@@ -64,11 +64,7 @@ const RecognitionPage = () => {
...
-              been diligently contributing to. This invitation can be from all
-              four github orgs of Layer5 depending on the number of
-              contributions you make.
+            Membership to the Github organizations is a significant milestone for contributors who have shown persistent commitment and dedication to Layer5 projects. It is not solely about writing code but rather the consistency of engagement and alignment of mentality. Those who have been actively involved in the community for several months, making updates, helping others, attending meetings, and demonstrating a willingness to learn and share their knowledge, are the ones who are likely to receive an invite. Such individuals demonstrate a genuine desire to improve themselves, others, and the projects they are working on. Depending on the number of contributions made across the five GitHub organizations of Layer5, contributors can expect to receive an invitation to any or all of these organizations. We value and appreciate the hard work and dedication of our contributors, and we are delighted to reward their efforts with access to our Github organizations
```
Comment (Chadha93) [inline src/sections/Community/Handbook/recognition.js:67]: Let's also mention the contributor recognition process - by Community Managers and Meshmates.

Sample - Community Managers (link it to community-managers page) and MeshMates (link it to the page) recognize potential contributors on a weekly basis, commonly inviting individuals after 6 to 8 weeks of continuous, active participation.

Comment (thisiskaransgit) [reply src/sections/Community/Handbook/recognition.js:67]: What you mentioned as a sample looks appropriate to me, will extend the paragraph by adding the following lines.

**Comment by l5io** [comment]
🚀 Preview for commit 6d1bb8cbe6e4da9e549364f8235fe464c90e61c9 at: https://64302f5703fb045bd386d805--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]
Thank you for adding about community managers and meshmates. LGTM

**Pull request merged**